app/services/project_reporter.rb

```diff
@@ -0,0 +1,40 @@
+class ProjectReporter
+  def initialize(project, reporter, description)
+    @project = project
+    @reporter = reporter
+    @description = description
+  end
+
+  def self.report(project:, reporter:, description:)
+    new(project, reporter, description).report
+  end
+
+  def report
+    gitter_client.send_message(text: message, room_id: ENV['MODERATORS_CHATROOM_ID'])
+    true
+  rescue => e
+    false
+  end
+
+  private
+
+  attr_reader :project, :reporter, :description
```

Comment:
The style we use in every other plain old ruby object is to have the attrs at the top of the class. Can we do the same here for consistency please?